src/PKSim.Core/Model/TransportDirection.cs

```diff
@@ -1,69 +1,190 @@
+
+      public static readonly TransportDirectionId[] MUCOSA_DIRECTIONS =
       {
-         return _allTransportDirections[transportDirectionId];
+         InfluxLumenToMucosaIntracellular,
+         EffluxMucosaIntracellularToLumen,
+         PgpMucosaIntracellularToLumen
+      };
+
+      public static readonly TransportDirectionId[] TISSUE_DIRECTIONS =
+      {
+         InfluxInterstitialToIntracellular,
+         EffluxIntracellularToInterstitial,
+         PgpIntracellularToInterstitial
+      };
+
+      public static readonly TransportDirectionId[] BRAIN_TISSUE_DIRECTIONS =
+      {
+         InfluxBrainInterstitialToTissue,
+         EffluxBrainTissueToInterstitial,
+         PgpBrainTissueToInterstitial
+      };
+
+      public static readonly TransportDirectionId[] BRAIN_BBB_DIRECTIONS =
+      {
+         InfluxBrainPlasmaToInterstitial,
+         EffluxBrainInterstitialToPlasma,
+         PgpBrainInterstitialToPlasma
+      };
+
+      public static TransportDirectionId DefaultVascularEndotheliumDirectionFor(TransportType transportType)
+      {
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxPlasmaToInterstitial;
+            default:
+               return EffluxInterstitialToPlasma;
+         }
       }
 
-      private static TransportDirection create(TransportDirectionId transportDirectionId, string displayName, string description, ApplicationIcon applicationIcon, bool canBeSetGlobally)
+      public static TransportDirectionId DefaultBloodCellsDirectionFor(TransportType transportType)
       {
-         var transportDirection = new TransportDirection(transportDirectionId, displayName, description,  applicationIcon, canBeSetGlobally);
-         _allTransportDirections.Add(transportDirection);
-         return transportDirection;
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxPlasmaToBloodCells;
+            default:
+               return EffluxBloodCellsToPlasma;
+         }
       }
 
-      public static IReadOnlyCollection<TransportDirection> All() => _allTransportDirections;
-   }
-   public class TransportDirection
-   {
-      public TransportDirectionId TransportDirectionId { get; }
-      public string DisplayName { get; }
-      public string Description { get; }
-      public ApplicationIcon Icon { get; }
-      public bool CanBeSetGlobally { get; }
+      public static TransportDirectionId DefaultMucosaDirectionFor(TransportType transportType)
+      {
+         switch (transportType)
+         {
+            case TransportType.Influx:
+               return InfluxLumenToMucosaIntracellular;
+            case TransportType.PgpLike:
+               return PgpMucosaIntracellularToLumen;
+            default:
+               return EffluxMucosaIntracellularToLumen;
+         }
+      }
+
+      public static TransportDirectionId DefaultTissueDirectionFor(TransportType transportType)
```

Comment:
All this logic to decide what to select for a transport type base on the location .I wanted to put that in the DB but it's much easier to have it as function here